shared/AppInsightsCore/src/JavaScriptSDK/CookieMgr.ts

```diff
@@ -288,6 +366,14 @@ export function createCookieMgr(rootConfig?: IConfiguration, logger?: IDiagnosti
             if (_isMgrEnabled(cookieMgr)) {
                 // Only remove the cookie if the manager and cookie support has not been disabled
                 result = cookieMgr.purge(name, path);
+            } else {
+                // Cache the deletion operation when cookies are disabled
+                // Format the deletion cookie string to use when cookies are re-enabled
+                _pendingCookies[name] = {
+                    o: ePendingOp.Purge,
+                    v: _formatCookieValue(STR_EMPTY, _createDeletionValues(path))
```

Comment:
As the only 2 places that this helper is called, lets update it to return the formatted deletion value and call the function _formatDeletetionValue